configs/nifi.properties

```diff
@@ -147,7 +147,7 @@ nifi.web.jetty.threads=200
 # nifi.web.proxy.context.path=
 
 # security properties #
-nifi.sensitive.props.key=
+nifi.sensitive.props.key={{ .Values.properties.sensitiveKey}}
 nifi.sensitive.props.key.protected=
 nifi.sensitive.props.algorithm=PBEWITHMD5AND256BITAES-CBC-OPENSSL
```

Comment:
I tested out these changes. I am on a brand new install of 1.14.0. I faced the following issue: https://www.mail-archive.com/users@nifi.apache.org/msg14466.html 

```
> Hi,
> according to NIFI-8246 defaults are
> 1.14.0:
> nifi.sensitive.props.algorithm=NIFI_PBKDF2_AES_GCM_256
> 1.13.2
> nifi.sensitive.props.algorithm=PBEWITHMD5AND256BITAES-CBC-OPENSSL
>
> The same passwords don't work
>
> My workaround was to export template from an old NiFi and to import it to
> a new one.
> Hope it helps )
>
> Alexey
```

So to get 1.14.0 to boot on a brand new install I had to change the setting to `nifi.sensitive.props.algorithm=NIFI_PBKDF2_AES_GCM_256` I don't know how nifi is going to react on an instance with previous data.

Reply:
Hi Drivard,

Thank you for these information. I will make these changes in the next update.